app/Kconfig

```diff
@@ -482,8 +494,12 @@ config ZMK_WPM
 	bool "Calculate WPM"
 	default n
 
-config SENSOR
+DT_COMPAT_ZMK_KEYMAP_SENSORS := zmk,keymap-sensors
```

Comment:
Does battery.c require `SENSOR` be enabled?

Reply:
Sure does. Added a new commit here that creates a new `ZMK_BATTERY_REPORTING` Kconfig symbol:

```
config ZMK_BATTERY_REPORTING
	bool "Battery level detection/reporting"
	default n
	select SENSOR
	select BT_BAS if ZMK_BLE
```

That is `imply`d by `ZMK_BLE` automatically.

Thoughts?